src/azmi-main/Operations.cs

```diff
@@ -44,16 +42,41 @@ public static string getToken(string endpoint = "management", string identity =
             }
         }
 
-        public static string getBlob(string blobURL, string filePath, string identity = null)
+        // Download the blob to a local file
+        public static string getBlob(string blobURL, string filePath, string identity = null, bool ifNewer = false)
         {
-            // Download the blob to a local file
+            // Connection
             var Cred = new ManagedIdentityCredential(identity);
             var blobClient = new BlobClient(new Uri(blobURL), Cred);
+
+            if (ifNewer)
+            {
+                var fileInfo = new System.IO.FileInfo(filePath);
+                if (fileInfo.Exists)

```

Comment:
use here static call to File.IfExist and use it inside of one if command, like
```csharp
if (ifNewer && File.Exists(filePath))
{
. . .
```

Reply:
Right.